### DISCUSSION TITLE
Make ruff server sort my imports (apply fixes) when formatting

### DISCUSSION THREAD

**Discussion by SichangHe** [astral-sh/ruff#12308] [Q&A]
The new `ruff server` native language server is nice. I am moving to it.

However, I looked around and could not find how to make it also sort imports on save.

Using NeoVim. Is this a setting I should specify at the language client, or is it something I should ask the folks at https://github.com/stevearc/conform.nvim for?

Thanks,\
Steven Hé (Sīchàng)

**Comment by SichangHe** [comment]
Well, this seems to work for LazyVim: https://github.com/SichangHe/.config/commit/f60646ee896dfc835c8e8f0d60ef5e01fc0a406f.

**Reply by dhruvmanila** [reply]
I'm not familiar with LazyVim but the LSP code looks correct to me. Feel free to ask any questions if it doesn't work.
